### ISSUE TITLE
Rimusic icon not always visible in android auto

### ISSUE THREAD

**Issue by Najib314** [fast4x/RiMusic#4870]
### Version of RiMusic

0.6.63.1

### Android version

13

### I've done these:

- [X] Uninstall old build before installing a new one
- [X] Checked opening issues about my similar problem

### Tell us how you trigger this bug

Plug device on android auto

### Submit a video

If you are on Android, please download a screen recorder and record your screen while you
duplicating the problem as describe above.

> You may need to use Github App to be able to upload video - size limit is 10 MB.

If you are on Desktop, drag video into this box and it will be uploaded automatically

### What was your expectation?

Have rimusic in android auto

**Comment by fast4x** [comment]
You mean icon or app ?

**Comment by Najib314** [comment]
Yep. Even if the app is launched on my phone, most of the time rimusic (icon and app) do not show in android auto. If I launch music from my phone it works, but I have no control

If I want them back I have to unplug and plug again several times the phone

**Comment by fast4x** [comment]
Look at this https://github.com/fast4x/RiMusic/wiki/FAQs#why-does-android-auto-not-see-rimusic

**Comment by orabidoo** [comment]
Seem blocked from google like other apps that streams YTmusic or spotify .

**Comment by Najib314** [comment]
I don confirm that when others apps like Spotify or YT Music are installed, theses apps seem to be prioritized. No problem since i uninstalled them. I had to reinstall YT Music, and the issue came back. Thank you again for your work
